types/react-native/index.d.ts

```diff
@@ -6800,7 +6800,7 @@ export interface BackHandlerStatic {
 
 export interface ButtonProps {
     title: string;
-    onPress: () => any;
+    onPress: (ev: any) => void;
```

Comment:
Could you use `NativeSyntheticEvent` or other explicit type instead of `any` here?